tests/raftstore/cluster.rs

```diff
@@ -163,10 +167,11 @@ impl<T: Simulator> Cluster<T> {
     }
 
     pub fn call_command(&self,
+                        node_id: u64,
```

Comment:
No, the reality is that we should always send the message to the specific store. Otherwise there should be some bugs.